libs/designer/src/lib/ui/panel/templatePanel/createWorkflowPanel/tabs/reviewCreateTab.tsx

```diff
@@ -75,8 +88,127 @@ export const ReviewCreatePanel = () => {
       id: 'OaUode',
       description: 'Accessibility label for no configuration required',
     }),
+    // Compact view strings
+    ACTION: intl.formatMessage({
+      defaultMessage: 'Action',
+      id: 'F9hn5p',
+      description: 'Label for action description',
+    }),
+    ACTION_NAME: intl.formatMessage({
+      defaultMessage: 'Your action name',
+      id: '4ZDZFr',
+      description: 'Label for action name',
+    }),
+    SUBSCRIPTION: intl.formatMessage({
+      defaultMessage: 'Subscription',
+      id: 'X/pilm',
+      description: 'Label for subscription',
+    }),
+    LOCATION: intl.formatMessage({
+      defaultMessage: 'Location',
+      id: 'OO37qq',
+      description: 'Label for location',
+    }),
+    RESOURCE_GROUP: intl.formatMessage({
+      defaultMessage: 'Resource group',
+      id: 'YAGa7I',
+      description: 'Label for resource group',
+    }),
+    AUTHENTICATION: intl.formatMessage({
+      defaultMessage: 'Authentication',
+      id: 'BHXsCs',
+      description: 'Label for authentication',
+    }),
+    PARAMETER: intl.formatMessage({
+      defaultMessage: 'Parameter',
+      id: '0/OIcB',
+      description: 'Label for parameter',
+    }),
+    VALUE: intl.formatMessage({
+      defaultMessage: 'Value',
+      id: 'YoMJq+',
+      description: 'Label for value',
+    }),
+    PRICING_ACKNOWLEDGEMENT: intl.formatMessage({
+      defaultMessage: 'I acknowledge that connecting to an Azure Logic Apps service will incur additional costs to my account.',
+      id: 'VRTo30',
+      description: 'Label for pricing acknowledgement',
+    }),
+    VIEW_PRICING: intl.formatMessage({
+      defaultMessage: 'View pricing',
+      id: 't/FNDI',
+      description: 'Label for view pricing',
+    }),
   };
 
+  const { data: subscriptions, isLoading: subscriptionLoading } = useSubscriptions();
+  const subscriptionDisplayName = useMemo(
+    () => subscriptions?.find((sub) => sub.id === `/subscriptions/${subscriptionId}`)?.displayName ?? '-',
+    [subscriptions, subscriptionId]
+  );
+
+  const styles = useStyles();
+
+  // Compact view
+  if (isCompactView) {
+    return (
```

Comment:
Can we use a different component here so we do not add strings for action and workflow mixed in here.
It would be great if this component can take in resourceOverrides, so if we need to use this here in azure portal we can call this workflowName or template name rather action name or action.